scss/main.scss

```diff
@@ -39,6 +45,10 @@ h2 {
   font-weight: 500;
 }
 
+hr {
+  border-color: #ccc !important;
```

Comment:
Please really avoid using `!important`! 😬 We can always create new rule targeting a more specific selector.

Reply:
😞 sorry, will remember that!